assignments/function-conversion.js

```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction =  () => {
 // console.log("Function was invoked!");
 // };
```

Comment:
or just in one line: 
`let myFunction = _ => console.log('Function was invoked!');`  :)